Content.Server/Access/IdCardConsoleSystem.cs

```diff
@@ -0,0 +1,80 @@
+using Content.Server.Access.Components;
+using Content.Shared.ActionBlocker;
+using Content.Shared.Verbs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Access
+{
+    public class IdCardConsoleSystem : EntitySystem 
+    {
+        [Dependency] private readonly ActionBlockerSystem _actionBlockerSystem = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<IdCardConsoleComponent, GetInteractionVerbsEvent>(AddInsertVerbs);
+            SubscribeLocalEvent<IdCardConsoleComponent, GetAlternativeVerbsEvent>(AddEjectVerbs);
+        }
+
+        private void AddInsertVerbs(EntityUid uid, IdCardConsoleComponent component, GetInteractionVerbsEvent args)
+        {
+            if (args.Using == null ||
+                !args.CanAccess ||
+                !args.CanInteract ||
+                !args.Using.HasComponent<IdCardComponent>() ||
+                !_actionBlockerSystem.CanDrop(args.User))
+                return;
+
+            // Can we insert a privileged ID? 
+            if (component.PrivilegedIDEmpty)
+            {
+                Verb verb = new("IDConsole:insertPrivileged");
```

Comment:
is it even strictly necessary?

Reply:
There is no actual requirement other than they should be unique. And yeah I was probably quite inconsistent with how I defined them while working on the PR. 

The keys are used to identify verbs when the client asks the sever to run a verb from the context menu that was defined server-side. So currently they are required, but maybe there is some better way of informing the server of what verb the client wants to run.

Previously when verbs were just classes these keys were generated automatically when the verbs were discovered while iterating over components using:
`$"{component.GetType()}:{verb.GetType()}"`
This lead to long (but at least guranteed to be unique) keys:
"Content.Server.Inventory.Components.InventoryComponent:Content.Server.Inventory.Components.InventoryComponent+SetOutfitVerb"

Now that verbs are not individual classes with unique names I don't know if there is a nice way of automatically generating unique keys. But at the very least what I've should be changed to be consistent.

Reply:
For now I'll just make it consistently use PascalCase with "[component/system]:[verbname/description]"

Reply:
is there any runtime verification/a test that they're unique?

Reply:
No there wasn't.  I've now made changes to remove verb keys altogether. Instead of using a `List<Verb>` & the `List.Sort()` function, it is now just using a `SortedSet<Verb>`. The "verb keys" are effectively implicitly defined by the verb `IComparable` function. 

If a new verb is equivalent according to the comparator, it will not be added to the set. Given that the comparator uses the verb text, icon, category and priority to sort the set of verbs, in most instances this should also guarantee that you never get entries in the context menu that appear identical.

This basically just shifts the "you should use unique keys" requirement to "you should use a unique verb text & category combination". Gven that requiring descriptive context menu entries is a sensible requirement that should be imposed anyways, I think that is fine.

Reply:
epic